mobile-app/app/screens/AppNavigator/screens/Settings/screens/ConfirmPinScreen.tsx

```diff
@@ -42,33 +44,39 @@ export function ConfirmPinScreen ({ route }: Props): JSX.Element {
     }
 
     const copy = { words, network, pin }
-    setSpinnerMessage(translate('screens/PinConfirmation', 'It may take a few seconds to update your passcode...'))
+    setSpinnerMessage(translate('screens/PinConfirmation', 'It may take a few seconds to update your passcode.'))
     setTimeout(() => {
       MnemonicEncrypted.toData(copy.words, copy.network, copy.pin)
         .then(async encrypted => {
           await MnemonicStorage.set(words, pin)
-          await setWallet(encrypted)
-          navigation.dispatch(StackActions.popToTop())
+          await passcodeChangeSuccess(encrypted)
```

Comment:
How would this await work if the function is not async?

Reply:
removed `await`